### PR TITLE
simplify startTask()

### PR DESCRIPTION
**Pull request by staabm**
| Q             | A
| ------------- | ---
| Bug fix?      | No
| New feature?  | No
| BC breaks?    | No
| Deprecations? | No
| Fixed tickets | N/A 

use the same structure we use in endTask(), with a early return.

this simplifies the IF condition.

### CODE REVIEW

File: src/Console/Output/Informer.php [hunk 1]
```diff
@@ -32,11 +32,12 @@ public function __construct(OutputWatcher $output)
 
     public function startTask(Task $task)
     {
+        if ($task->isShallow()) {
+            return;
+        }
+
         $this->startTime = round(microtime(true) * 1000);
```
Comment (antonmedv) [inline src/Console/Output/Informer.php]: But we loose startTime here.

Comment (staabm) [reply src/Console/Output/Informer.php]: we use startTime only in one place inside this file and this case has the same IF guard

Comment (staabm) [reply src/Console/Output/Informer.php]: How to proceed?

**Comment by staabm** [comment]
the few lines change is not worth discussing.

**Pull request closed without merging**